opsimate-docs/docs/alerts/actions.md

```
@@ -0,0 +1,59 @@
+| `{{summary}}` | Alert summary |
+| `{{service}}` | Service the alert relates to |
+| `{{startsAt}}` | When the alert started |
+| `{{createdAt}}` | When the alert was created |
+| `{{updatedAt}}` | When the alert was last updated |
+| `{{url}}` | Link to the alert in OpsiMate |
+| `{{runbookUrl}}` | Link to the runbook, if set |
+
+### Tags
+Tag values are available under three prefixes, all equivalent:
+ 
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)